scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -594,6 +595,56 @@ def add_entry_to_logrotate_conf_file(self):
         for line in lines:
             fp.write(line)
 
+def update_btree_watermarks(self):
+    services_limits = Conf.get(self._index, 'cortx>motr>limits')['services']
+    try:
+        cvg = self.storage['cvg']
+    except:
+        raise MotrError(errno.EINVAL, "cvg not found\n")
+    # Check if cvg type is list
+    check_type(cvg, list, "cvg")
+    max_mem_limit_for_ios = 0
+    min_mem_limit_for_ios = 0
+
+    for arr_elem in services_limits:
+        if arr_elem['name'] == "ios":
+            l_min = arr_elem['memory']['min']
+            if l_min.isnumeric():
+                min_mem_limit_for_ios = int(l_min)
+            else:
+                min_mem_limit_for_ios = calc_size(self, l_min)
+
+            l_max = arr_elem['memory']['max']
+            if l_max.isnumeric():
+                max_mem_limit_for_ios = int(l_max)
+            else:
+                max_mem_limit_for_ios = calc_size(self, l_max)
+
+    wm_low  = min_mem_limit_for_ios
+    wm_targ = int(max_mem_limit_for_ios * 0.60)
+    wm_high = int(max_mem_limit_for_ios * 0.85)
+
+    self.logger.info(f"setting MOTR_M0D_BTREE_LRU_WM_LOW to {wm_low}\n")
+    cmd = f'sed -i "/MOTR_M0D_BTREE_LRU_WM_LOW/s/.*/MOTR_M0D_BTREE_LRU_WM_LOW={wm_low}/" {MOTR_SYS_CFG}'
+    execute_command(self, cmd)
```

Comment:
Maybe we can define a function to avoid duplication similar code here and followings.

Reply:
Took care in PR-1966